tests/fixtures/integration/make_Bos2016_data/meanfield/Makefile

```
@@ -0,0 +1,15 @@
+
+all:	kummers_function.so
+
+kummers_function.so:	kummers_function.pyf kummers_function.f90
+	f2py3 -c kummers_function.pyf -lmathlib kummers_function.f90
+#	f2py -c kummers_function.pyf -L/home/helias/slatec-4.1 -lslatec kummers_function.f90
+
+
+
+kummers_function.pyf: 	kummers_function.f90
+	f2py3 kummers_function.f90 -m kummers_function -h kummers_function.pyf
```

```suggestion
	f2py -c kummers_function.pyf -lmathlib kummers_function.f90



kummers_function.pyf: 	kummers_function.f90
	f2py kummers_function.f90 -m kummers_function -h kummers_function.pyf
```

I noticed that the with Bos2016_env.yaml installing python 2.7, the fortran_functions, need to be created for python 2 as well. If called with 'make', the created fortran_function.so does work.

In #34 the folder structure was different (/lif_meanfield_tools/lif_meanfield_tools/tests/integration...) and I think, when porting to the new structure you missed the .so/.pyc file which are checked in (in short: they are still in the old place).

---

Unfortunately, I don't remember how Fortran works (although I once had to work with it). Can you change what needs to be changed, or does the suggested change already solve the problem?